src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -107,15 +89,9 @@
 import net.pms.util.XMLRPCUtil.MemberString;
 import net.pms.util.XMLRPCUtil.Params;
 import net.pms.util.XMLRPCUtil.Struct;
```

Comment:
When XMLRPCUtil is imported static those three lines are not needed to be here.

Reply:
@valib You're looking at an outdated commit. I had to do some merging to combine mine and @SubJunk's changes, I thing that comes from there.

Reply:
@valib I checked, and this merge error were still in the updated commits. I've fixed it and pushed now. Thanks :+1: